include/ppx/application.h

```diff
@@ -464,15 +464,15 @@ class Application
 
     // Returns true when a run is active, otherwise returns false.
     // See StartMetricsRun for why this wrapper is necessary.
-    bool HasActiveMetricsRun() const;
+    virtual bool HasActiveMetricsRun() const;
```

Comment:
You should add a comment to say "it's virtual for unit testing, even if it's not visible here"
Otherwise we'll probably end up cleaning this out thinking it's virtual for no reason.

Reply:
Done.